docs/topics/rest-http-README.adoc

```diff
@@ -0,0 +1,80 @@
+
+:runtime: ${runtime}
+
+ifeval::["{runtime}" == "Spring Boot"]
+:localRunCMD: mvn spring-boot:run
+:OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
+:guideURL: http://appdev.openshift.io/docs/spring-boot-runtime.html
+:fileLocation: src/main/resources/static/index.html
+endif::[]
+
+ifeval::["{runtime}" == "Eclipse Vert.x"]
+:localRunCMD: mvn vertx:run
+:OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
+:guideURL: http://appdev.openshift.io/docs/vertx-runtime.html
+:fileLocation: src/main/resources/webroot/index.html
+endif::[]
+
+ifeval::["{runtime}" == "WildFly Swarm"]
+:localRunCMD: mvn wildfly-swarm:run
+:OSORunCMD: mvn clean fabric8:deploy -Popenshift -DskipTests
+:guideURL: http://appdev.openshift.io/docs/wf-swarm-runtime.html
+:fileLocation: src/main/webapp/index.html
+endif::[]
+
+
+
+= ${mission} - ${runtime} Booster
+
+== Run this Booster Locally
+
+To run this booster on your local host:
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ git clone git@github.com:${loggedUser}/${targetRepository}
+
+$ cd ${targetRepository}
+
+$ {localRunCMD}
+----
+
+== Interact with this Booster Locally
+
+To interact with your booster while its running, use the form at `http://localhost:8080` or the `curl` command:
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ curl http://localhost:8080/api/greeting
+{"content":"Hello, World!"}
+
+$ curl http://localhost:8080/api/greeting?name=Sarah
+{"content":"Hello, Sarah!"}
+----
+
+
+== Update this Booster
+To update your booster:
+
+. Stop your booster.
+. Make your change (e.g. edit link:https://github.com/${loggedUser}/${targetRepository}/edit/master/{fileLocation}[`{fileLocation}`]).
+. Restart your booster.
+. Confirm your change appears.
+
+
+== Run this Booster on a Single-node OpenShift Cluster
+If you have a single-node OpenShift cluster, such as Minishift or the Red Hat Container Development Kit, link:http://appdev.openshift.io/docs/minishift-installation.html[installed and running], you can also deploy your booster there. A single-node OpenShift cluster provides you with access to a cloud environment that is similar to a production environment than simply running directly on your local host.
```

Comment:
Extra comma in "Minishift or the Red Hat Container Development Kit, installed and running".

"access to a cloud environment that is similar to a production environment than simply running directly on your local host" sounds weird. Maybe something like "access to a cloud-like environment that is more similar to a production environment than simply running directly on your local host"?